### PR TITLE
Update contributing guide with code of coduct

### PR DESCRIPTION
**Pull request by baijum**
Ref. https://issues.redhat.com/browse/APPSVC-787

### CODE REVIEW

**Comment by sbose78** [comment]
Thank you Baiju.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,6 +7,101 @@
...
+Note: If your git config information is set properly then viewing the `git log`
+information for your commit will look something like this:
+
+```
+Author: Joe Smith <joe.smith@example.com>
+Date:   Thu Feb 2 11:41:15 2018 -0800
+
+    Update README
+
+    Signed-off-by: Joe Smith <joe.smith@example.com>
```
Comment (pedjak) [inline CONTRIBUTING.md:98]: we should add CI job checking this, preferably in this PR.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,6 +7,101 @@
   * [Suggesting Enhancements](#suggesting-enhancements)
   * [Pull Requests](#pull-requests)
```
Comment (pedjak) [inline CONTRIBUTING.md:8]: some links are dead, please verify - should we update TOC, or remove it?

File: CONTRIBUTING.md (hunk 1 above)
Comment (baijum) [reply CONTRIBUTING.md:98]: Let's see, if we can enable this: https://probot.github.io/apps/dco/

Comment (pedjak) [reply CONTRIBUTING.md:98]: it looks it is configured now?

Comment (baijum) [reply CONTRIBUTING.md:98]: Yes. See this new PR #834

File: CONTRIBUTING.md (hunk 2 above)
Comment (baijum) [reply CONTRIBUTING.md:8]: I would suggest removing TOC. It's not really required for a doc like this.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,6 +7,101 @@
...
 
+:+1::tada: First off, thanks for taking the time to contribute!
+
+You can look at the issues [with help wanted label][help-wanted] for items that
+you can work on.
+
+If you need help, please feel free to reach out to our [slack channel](https://app.slack.com/client/T09NY5SBT/C019LQYGC5C)!
```
Comment (DhritiShikhar) [inline CONTRIBUTING.md:15]: This link is not correct. It should point to kubernetes workspace

Screenshot from 2021-01-07 17-27-03

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -7,6 +7,101 @@
...
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+
+Then you just add a line to every git commit message:
```
Comment (DhritiShikhar) [inline CONTRIBUTING.md:80]: This should also be mentioned:

`git commit --signoff`

File: CONTRIBUTING.md (hunk 3 above)
Comment (baijum) [reply CONTRIBUTING.md:15]: If the person already logged into Kubernetes slack. That link will open the channel.

File: CONTRIBUTING.md (hunk 4 above)
Comment (baijum) [reply CONTRIBUTING.md:80]: That will require more explanation about the Git configuration file also. We can assume the developer knows Git.

File: CONTRIBUTING.md (hunk 2 above)
Comment (baijum) [reply CONTRIBUTING.md]: removed TOC

**Pull request merged**